### ISSUE TITLE
Hi

### ISSUE THREAD

**Issue by Rishiroshan** [RDXLR/RDXLR#1]
Please can you help me if some one is using my phone remote please

**Comment by Fuxroundfinout** [comment]
I need help please can I contact you or some how you contact me

**Comment by Alygnt** [comment]
> I need help please can I contact you or some how you contact me

Please drop me a email to mralygnt@gmail.com or DM me in insta ID: mralygnt

**Comment by Rishiroshan** [comment]
Yeah please do let me know Sent from my iPhoneOn 20 Oct 2023, at 19:29, Fuxroundfinout ***@***.***> wrote:﻿
I need help please can I contact you or some how you contact me
